.env-example

```diff
@@ -0,0 +1,6 @@
+# These database variables establishes a connection between the application and your local database setup
+
+DATABASE_NAME=your database name
```

Comment:
Alternatively, you could also have used `DATABASES = {
    'default': dj_database_url.config(default=config("DATABASE_URL"))
}`...this would reduce the database url to `postgresql://YOUR_ROLE:PASSWORD@localhost:5432/DB_NAME`. Otherwise great job.

Reply:
That is a good idea but we would have to install the `dj-database-url` package which adds unnecessary overhead.